app/helpers/no_cms/blocks/blocks_helper.rb

```diff
@@ -21,7 +21,7 @@ def render_block_slot block_slot, options= {}
         #check if block is lazy
         options[:lazy_block] = !options[:lazy_block].nil? ? options[:lazy_block] : block_slot.template_zone_config.
                                 config[:lazy_blocks].
-                                include?(block_slot.block.template)
+                                include?(block_slot.block.template) rescue false #Si no tenemos el atributo lazy block lanza excepción
```

Comment:
We should avoid rescues, it's not a best practice. We should copy what's done in https://github.com/simplelogica/nocms-blocks/blob/features/landing-promo/app/models/no_cms/blocks/layout.rb#L41

We have a `DEFAULT_FIELD_CONFIGURATION` attribute (that's not been used right now) where we should put default configuration (e.g an empty array for blocks and lazy blocks) and then merge it so the config always have a lazy_blocks attribute and a blocks attribute (and whatever is needed)